apps/address-service/domains/user/schema/User.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Generated by `createschema user.User 'name:Text;isAdmin:Checkbox;email:Text;password:Password;'`
+ */
+
+const { Text, Checkbox, Password } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@condo/keystone/plugins')
+const { dvAndSender } = require('@address-service/domains/common/schema/plugins/dvAndSender')
+const access = require('@address-service/domains/user/access/User')
+
+
+const User = new GQLListSchema('User', {
+    // TODO(codegen): write doc for the User domain model!
```

Comment:
it's better to copy the model from miniapp

Reply:
We don't need all miniapp user's fields for the model in the address service.